pkg/cable/ipsec/strongswan.go

```diff
@@ -34,8 +36,18 @@ const (
 
 	// charonViciSocket points to the vici socket exposed by charon
 	charonViciSocket = "/var/run/charon.vici"
+
+	cableDriverName = "ipsec/strongswan"
 )
 
+func init() {
+	cable.SetDefautCableDriver(cableDriverName)
+	cable.AddDriver(cableDriverName, NewStrongSwan)
+	cable.AddDriver("ipsec", NewStrongSwan)
+	cable.AddDriver("strongswan", NewStrongSwan)
```

Comment:
On the call today, it was decided to keep it simple and register the exact driver name sans aliases/defaults and sans _"ipsec"_ prefix.  So driver name would just be _"strongswan"_.

Reply:
Done, please review

Reply:
> On the call today, it was decided to keep it simple and register the exact driver name sans aliases/defaults and sans _"ipsec"_ prefix. So driver name would just be _"strongswan"_.

Sorry, I missed this comment.

I think it should also be fine to go with cabledriver, and that's it ;)